DevCycle/DVCUser.swift

```diff
@@ -44,6 +44,10 @@ public class DVCUser: Codable {
         self.deviceModel = platform.deviceModel
         self.sdkType = platform.sdkType
         self.sdkVersion = platform.sdkVersion
+        self.appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
+        if let appBuildStr = Bundle.main.infoDictionary?["CFBundleVersion"] as? String {
+            self.appBuild = Int(appBuildStr)
```

Comment:
Seems like it is a string to the API, let me play around with it: https://github.com/taplytics/DevCycle/blob/main/cf-workers/client-sdk-api/docs/client-sdk-api.yaml#L222

Reply:
yea seems like our API expects to save it as a number. I'll just keep it like this, and the casting of the `Int()` will fail and it will just keep it as a `nil` value.

Reply:
example app still runs with it like this: 
![image](https://user-images.githubusercontent.com/1219069/151056310-d53114ee-ab65-490f-9c32-8d9ce99a1af0.png)